windows/CodePushReactPackage.cs

```diff
@@ -1,25 +1,18 @@
 ﻿using Newtonsoft.Json.Linq;
-using Newtonsoft.Json;
 using ReactNative;
 using ReactNative.Bridge;
 using ReactNative.Modules.Core;
 using ReactNative.UIManager;
 using System;
 using System.Collections.Generic;
-using System.Reflection;
 using System.Threading.Tasks;
-using Windows.ApplicationModel;
-using Windows.Storage;
-using Windows.Web.Http;
+
 
 namespace CodePush.ReactNative
 {
     public sealed class CodePushReactPackage : IReactPackage
     {
         private static CodePushReactPackage CurrentInstance;
-        private static bool NeedToReportRollback = false;
-
```

Comment:
It was part of @geof90 's original commit (according to git blame). The value is set to true, but never read from. Looks like it was transliterated in to C# incorrectly, maybe? We can file and issue and do a separate PR to fix -- we specifically have user stories in our project for update failure modes that will likely force us to fill missing gaps on that front.